src/components/Toolbar.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <ion-header :translucent="true">
+    <ion-toolbar>
+      <div id="title-bar" >
+        <ion-title>Product Order</ion-title>
+        <ion-title id="middle-title">{{titleText}}</ion-title>
+        <ion-button color="primary">Login</ion-button>
```

Comment:
Correct me if I'm wrong, but you're using this in the pages as the toolbar right? but there's this static button with Login label on it. What's the purpose of this? Same with the double ion-title tags.

Reply:
I plan to have the button switch to a label of the logged in user once the back end implemented. The second title tag changes between pages.